src/start/AuthScreen.js

```diff
@@ -31,11 +31,8 @@ class AuthScreen extends React.Component {
           keyboardShouldPersistTaps="always"
         >
           <View style={styles.container}>
-            <Text style={[styles.field, styles.heading1]}>
-              Welcome to Zulip!
-            </Text>
-            <Input
-              customStyle={[styles.field, styles.disabled]}
+            <Label
+              style={[styles.field]}
```

Comment:
Label not having text ultimately FormattedMessage not having ID throwing an error